index.js

```diff
@@ -0,0 +1,12 @@
+const context = require.context("./docs/tools/", true, /\.md$/);
```

Comment:
I only need to pull markdown files from the `tools` directory (and below, naturally), at least for now.